app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,77 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+before_filter :sign_up_params, only: [:create]
+# before_filter :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  def new
+    @user = User.new
+  end
+
+  # POST /resource
+  def create
```

Comment:
with such approach user will be registered and be able to login to application
so it gives us nothing except "wait for approve" message